src/main/features/claim/views/new-features-consent.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, submitButton %}
+
+{% set heading = 'Do you want to try new features as we add them?' %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <p>{{ t('We’re continuously adding new features to this digital service. You can choose to try these features as we launch them') }}</p>
+    <form method="POST" novalidate class="analytics-submit-event-trigger" data-event-label="New features consent">
```

Comment:
We need `data-event-label-from="consentResponse"` as well to make it work otherwise JS does not know what to take data from.